src/widget/form/chatform.cpp

```diff
@@ -1025,8 +1025,7 @@ void ChatForm::onExportChat()
     QDateTime now = QDateTime::currentDateTime();
     QList<History::HistMessage> msgs = history->getChatHistory(pk, epochStart, now);
 
-    QString path = QFileDialog::getSaveFileName(0, tr("Save chat log"), QString{}, QString{}, 0,
-                                                QFileDialog::DontUseNativeDialog);
```

Comment:
Why do you remove `DontUseNativeDialog`? It was added in #3447 to fix issue with some file managers